elfi/examples/bdm.py

```diff
@@ -57,6 +58,7 @@ def process_result(completed_process, *inputs, **kwinputs):
     os.remove(output_filename)
 
     # This will be passed to ELFI as the result of the command
+    print("Test 4")
```

Comment:
another test print that maybe should be removed?

Reply:
Yes, thanks!